components/compliance-service/ingest/ingestic/ingestic.go

```diff
@@ -1064,3 +1064,116 @@ func (backend *ESClient) SetDailyLatestToFalseForControlIndex(ctx context.Contex
 		Do(ctx)
 	return errors.Wrap(err, "SetDailyLatestToFalseForControlIndex")
 }
+
+func (backend *ESClient) GetNodesDayLatestTrue(ctx context.Context) ([]relaxting.NodesUpgradation, error) {
+	var nodes []relaxting.NodesUpgradation
+	time90daysAgo := time.Now().Add(-24 * time.Hour * 90)
+	indices, err := GetLast90DaysIndices(time90daysAgo, time.Now())
+	if err != nil {
+		return nil, err
+	}
+
+	boolQuery := elastic.NewBoolQuery().
+		Must(elastic.NewTermQuery("day_latest", true)).
+		Must(elastic.NewTermQuery("daily_latest", true))
+
+	fsc := elastic.NewFetchSourceContext(true).Include(
+		"node_uuid",
+		"end_time",
+		"day_latest",
+	)
+
+	searchSource := elastic.NewSearchSource().
```

Comment:
I think 1000 is not correct number of records for last 90 days. If we want to collect all the nodes in a single query. I don't think its possible